reference_controllers/rabbitmq_controller_unverified/src/erlang_cookie.rs

```diff
@@ -0,0 +1,46 @@
+use k8s_openapi::api::apps::v1 as appsv1;
+use k8s_openapi::api::core::v1 as corev1;
+use k8s_openapi::apimachinery::pkg::api::resource::Quantity;
+use k8s_openapi::apimachinery::pkg::apis::meta::v1 as metav1;
+use k8s_openapi::ByteString;
+use kube::{
+    api::{Api, ListParams, PostParams},
+    runtime::controller::{Action, Controller},
+    Client, CustomResourceExt,
+};
+use kube_client::{self, client};
+use kube_core::{self, Resource};
+use rand::Rng;
+use std::{collections::BTreeMap, vec};
+
+use crate::rabbitmqcluster_types::RabbitmqCluster;
+
+pub fn erlang_build(rabbitmq: &RabbitmqCluster) -> corev1::Secret {
+    let cookie = "aaaaaaaaaaaaaaaaaaaaaaa".to_string().into_bytes();
```

Comment:
Is it ok to use such a random cookie?

Reply:
If they recommend to generate the cookie using random number then you should do so

Reply:
Originally, the generate_cookie is used to generate cookie. But I found it sometimes generate cookie with invalid characters and cause the failure of building image. I will try to modify it to make sure the cookie is correctly and randomly generated.